Zend/zend_builtin_functions.c

```diff
@@ -1317,114 +1317,63 @@ ZEND_FUNCTION(property_exists)
 }
 /* }}} */
 
-/* {{{ proto bool class_exists(string classname [, bool autoload])
-   Checks if the class exists */
-ZEND_FUNCTION(class_exists)
+static inline void class_exists_impl(INTERNAL_FUNCTION_PARAMETERS, int flags, zend_bool expect) /* {{{ */
```

Comment:
For symmetry with get_declared_class_impl I'd use flags + skip_flags here as well, instead of flags + expect.